account_invoice_validate_send_email/models/mail_compose_message.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailComposeMessage(models.TransientModel):
+    _inherit = "mail.compose.message"
+
+    @api.multi
+    def get_mail_values(self, res_ids):
+        self.ensure_one()
+        results = super(MailComposeMessage, self).get_mail_values(res_ids)
+        reply_to = self.reply_to
+        for res_id in res_ids:
+            results[res_id]["reply_to"] = reply_to
```

Comment:
I understood there is a case that `self.reply_to` is False. Now, I look back my work and thought until yesterday, I faced the case repeatedly because I failed to recognize requirement of template settings of reply_to.
I'll change the codes soon.